WordPress/src/test/java/org/wordpress/android/ui/uploads/UploadActionUseCaseTest.kt

```diff
@@ -109,11 +110,11 @@ class UploadActionUseCaseTest {
         val siteModel: SiteModel = createSiteModel()
 
         // Act and Assert
-        assertThat(posts).allSatisfy { post ->
+        assertThat(posts).allSatisfy (Consumer { post ->
```

Comment:
Minor (🔍): See suggested change below.

```suggestion
        assertThat(posts).allSatisfy(Consumer { post ->
```

Reply:
If I am not mistaken, this is how I initially typed this line then asked AS to format it. I don't mind one or the other, so feel free to include a commit about this in one of your PRs.

Reply:
> If I am not mistaken, this is how I initially typed this line then asked AS to format it.

Actually, you updated 2 files with such change

- On this [FetchScanStateUseCaseTest](https://github.com/wordpress-mobile/WordPress-Android/pull/17618/files#diff-f58b7d2038a1ac3cf30d988c363ad52f14a87af5ea4704947df9b58e94522dbeR152) test you used the `assertThat(result).allSatisfy(Consumer { it is Success })` syntax, but
- On this [UploadActionUseCaseTest](https://github.com/wordpress-mobile/WordPress-Android/pull/17618/files#diff-818c082675aff846226e7e0a82b3267aa9f951650733d8ff3bda1a428506160dR113) you used the `assertThat(posts).allSatisfy( Consumer { post -> ... }`. 

Thus, seeing that, I opened AS and tried to auto-format both to see which one we should keep, the end result was that the `allSatisfy(Consumer` was the preferred format instead of the `allSatisfy( Consumer` one.

I hope that clarified it @oguzkocer ... 🤔 

> I don't mind one or the other, so feel free to include a commit about this in one of your PRs.

Thus, I took the liberty to comment on that, suggest a change and actually act on that on your behalf since you were aways, it being minor and all. I hope that's okay. 🙏

FYI: There is no need to included that change in any of my PRs as this change was already merged as part of this PR.

Reply:
That makes sense. I remember formatting the string, but I guess I did for one and not the other. Sorry about that! 🤦 

I also completely missed the new commits, and saw that the PR was merged, so I suggested the change to be included in one of your PRs. Again, sorry about that.

_P.S: This is what I get for trying to squeeze in some work while I am AFK 😞_

Reply:
😄 and 🤗 Oguz, I am sorry if I  unintentionally drag you into all that... 🙏

Please rest now, we will catch-up on all that on your return! ❤️

Reply:
Not at all. I am just trying to ease my workload for when I return tomorrow. It's better for me to work in short bursts, but as a result, I am not fully focused on the work. In any case, thanks for understanding ❤️